ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3982,6 +3982,20 @@ AHU_SFSS_SFMSC_SFVSC_FDPM_CHWSC_RTM_RACO2C_SSPC2X_VOADM2X_SAIDC2X_RAIDC2X_SSPC:
   - RAIDC2X
   - SSPC
 
+AHU_HTSC_VOADM_ETM_RMM_FDPM2X_RSM_SEFC_FAM_HRC:
+  description: "Unit with temperature monitoring, differential pressure sensors and heat recovery control."
+  is_canonical: true
+  implements:
+  - AHU
+  - HTSC
+  - VOADM
```

Comment:
This is used when you have a device that mixes return with outside air, usually for economizing purposes. The rest of the context in this PR indicates that what you have is more of a single-pass DOAS system. Can you please confirm the configuration of this device and how it functions? (`VOADM` defines `mixed_air_temperature_sensor`, which is only appropriate for systems that can recirculate air from the space, mixing it with outside air).